classes/core/PKPBladeViewServiceProvider.php

```diff
@@ -116,13 +114,15 @@ function (\Illuminate\View\Component $component, string $viewPath): string {
             ?>";
         });
 
-        // Register the sanitizeHtml macro
-        Str::macro('sanitizeHtml', function (string $input, string $configKey = 'allowed_html') {
```

Comment:
As this is getting moved to `ViewHelper`, need to update the basic doc at https://github.com/pkp/pkp-lib/issues/9968 in issue description .